io/plugins/eu.esdihumboldt.hale.io.shp/src/eu/esdihumboldt/hale/io/shp/writer/ShapefileInstanceWriter.java

```diff
@@ -138,7 +138,7 @@ protected String getDefaultTypeName() {
 	 * 
 	 */
 
-	protected void writeInstances(InstanceCollection instances, ProgressIndicator progress,
+	public void writeInstances(InstanceCollection instances, ProgressIndicator progress,
```

Comment:
Not sure if it's necessary to change the visibility of this method. Wouldn't it be possible instead to use the `ShapefileInstanceWriter` by calling its `execute()` method?